metrics/prometheus/prometheus.go

```diff
@@ -0,0 +1,302 @@
+package metrics
```

Comment:
Do you think it makes sense to 'preload' the values as we do for Prebid Server to avoid the initial setup performance hit

Reply:
Stole your preload functions from `prebid-server`. But, to my understanding, only `CounterVec`s and `HistogramVec`s need to be preloaded, right? Do I need to do something about non-vector `Histogram` and `Counter` metrics?

Reply:
Yes, that's right. No preload is needed for non-vector `Histogram` and `Counter` metrics.